Marlin/example_configurations/delta/biv2.5/Configuration.h

```diff
@@ -538,6 +543,11 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = true; // set to true to invert the lo
   #define Z_PROBE_ALLEN_KEY_STOW_3_Z 100.0
   #define Z_PROBE_ALLEN_KEY_STOW_3_FEEDRATE XY_PROBE_SPEED
 
+  #define Z_PROBE_ALLEN_KEY_STOW_4_X 0.0
+  #define Z_PROBE_ALLEN_KEY_STOW_4_Y 0.0
+  #define Z_PROBE_ALLEN_KEY_STOW_4_Z Z_PROBE_ALLEN_KEY_STOW_3_Z
+  #define Z_PROBE_ALLEN_KEY_STOW_4_FEEDRATE XY_PROBE_SPEED
```

Comment:
```
 -      destination[Z_AXIS] = Z_PROBE_ALLEN_KEY_STOW_3_Z;		
 -      prepare_move_to_destination();		
 -		
 -      // Home XY for safety		
 -      feedrate = homing_feedrate[X_AXIS] / 2;		
 -      destination[X_AXIS] = 0;		
 -      destination[Y_AXIS] = 0;		
 -      prepare_move_to_destination(); // this will also set_current_to_destination		
 -		
 -      feedrate = old_feedrate;		
```